### ISSUE TITLE
Desktop: Keyboard focus not visible

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#9982]
### Operating system

Linux

### Joplin version

2.14.15

### Desktop version info

Joplin 2.14.15 (dev, linux)

Client ID: 44da07c87ebf4662aa0e0f97f77f7f38
Sync Version: 3
Profile Version: 46
Keychain Supported: No

Revision: 3e1790af1 (pr/allow-cm6-user-commands-to-take-arguments)

CodeMirror6 line numbers: 1.0.0
Simple Backup: 1.3.5

### Current behaviour

1. Press <kbd>tab</kbd> to change the focus (**hint**: <kbd>Escape</kbd> then <kbd>tab</kbd> can be used to move focus out of the beta editor).
2. Try to determine what is focused visually.

> [!NOTE]
>
> This is particularly an issue when no mouse is connected to a computer. If focus indicators were originally removed for stylistic/appearance reasons, I suggest enabling them if no `pointermove` events have been received.
>
> Alternatively, we may be able to let Electron decide whether to show focus indicators with the [`:focus-visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) CSS selector.
>

### Expected behaviour

It should be possible to visually determine what is focused when navigating using the keyboard. This is an issue particularly when no mouse/pointer device is connected to a computer.

See https://hacks.mozilla.org/2019/06/indicating-focus-to-improve-accessibility/ for why visible keyboard focus is important and implementation suggestions.
